test/python/providers/test_fake_backends.py

```diff
@@ -60,8 +60,8 @@ def test_circuit_on_fake_backend(self, backend, optimization_level):
             self.circuit,
             backend,
             optimization_level=optimization_level,
-            seed_simulator=42,
-            seed_transpiler=42,
+            seed_simulator=4042,
+            seed_transpiler=4042,
```

Comment:
This seems like it's telling a story, haha.

Reply:
Lol, I actually need to check if it's still needed. It was necessary before #7276 because the vf2 layout was picking was picking the worst 2 qubits (because they were higher connectivity) on `FakeYorktown` and the default seed was causing the most likely result in the counts object to be incorrect with the error rates when running with aer. But I need to check that after #7276 fixed the bias in the VF2Layout towards qubits with higher noise this isn't needed.

Reply:
I undid this in: https://github.com/Qiskit/qiskit-terra/pull/7213/commits/37aabdc018940518c72add2eadc7e526cd89e49b as with all the improvements in #7276 this wasn't needed anymore because with `FakeYorktown` the minimal noise awareness of the pass knows to not use the super noisy qubits